src/ServiceInsight.Tests/MessageListViewModelTests.cs

```diff
@@ -28,39 +29,40 @@ public class MessageListViewModelTests
         public void TestInitialize()
         {
             eventAggregator = Substitute.For<IEventAggregator>();
+            workNotifier = Substitute.For<IWorkNotifier>();
             serviceControl = Substitute.For<IServiceControl>();
             searchBar = Substitute.For<SearchBarViewModel>();
             clipboard = Substitute.For<IClipboard>();
-            messageListFunc = () => new MessageListViewModel(eventAggregator,
-                                                   serviceControl,
-                                                   searchBar,
-                                                   Substitute.For<GeneralHeaderViewModel>(),
-                                                   Substitute.For<MessageSelectionContext>(),
-                                                   clipboard);
+            messageListFunc = () => new MessageListViewModel(
+                eventAggregator,
+                workNotifier,
+                serviceControl,
+                searchBar,
+                Substitute.For<GeneralHeaderViewModel>(),
+                Substitute.For<MessageSelectionContext>(),
+                clipboard);
         }
 
         [Test]
         public void Should_load_the_messages_from_the_endpoint()
         {
             var endpoint = new Endpoint { Host = "localhost", Name = "Service" };
             serviceControl.GetAuditMessages(Arg.Is(endpoint), Arg.Any<string>(), Arg.Any<int>(), Arg.Any<string>(), Arg.Any<bool>())
-                             .Returns(x => new PagedResult<StoredMessage>
-                             {
-                                 CurrentPage = 1,
-                                 TotalCount = 100,
-                                 Result = new List<StoredMessage>
-                                 {
-                                     new StoredMessage(),
-                                     new StoredMessage()
-                                 }
-                             });
+                .Returns(x => new PagedResult<StoredMessage>
+                {
+                    CurrentPage = 1,
+                    TotalCount = 100,
+                    Result = new List<StoredMessage>
+                    {
+                        new StoredMessage(),
+                        new StoredMessage()
+                    }
+                });
```

Comment:
Could we change the return result so it makes sense? There `100` total count is clearly incorrect!